.packit.yml

```diff
@@ -0,0 +1,74 @@
+---
+actions:
+  post-upstream-clone:
+  - >-
+    python3 -m
+    pip install setuptools_scm tox
+    --user
+  create-archive:
+  - >-
+    bash -c '
+    echo %upstream_version "${PACKIT_PROJECT_VERSION}"
+    |
+    tee ~/.rpmmacros
+    '
```

Comment:
@TomasTomecek could you help me with this? I've been meaning to define an in-job user-local macro for `rpmbuild` to consume it so I wouldn't really have to let `fix-spec-file` modify the spec file.
Am I right that it's https://github.com/packit/specfile that fails to parse it properly? It's unclear from the logs. Any ideas?

Reply:
Debugged it, and it's coming from the unsanitized/unescaped input crashing the call to `rpm.spec()`. At least, partially.

Reply:
Logged it at https://github.com/packit/packit-service/issues/1659. Although, that error isn't exactly the same as what's visible in CI and maybe there's more.